libraries/AP_IOMCU/iofirmware/ioprotocol.h

```diff
@@ -143,6 +143,9 @@ struct page_mixing {
     uint8_t rc_reversed[IOMCU_MAX_CHANNELS];
     uint8_t rc_channel[4];
 
+    // enabled RC protocols
+    uint32_t rc_protocols;
```

Comment:
Should this default to `1`?  So until we're told otherwise by the FMU we assume all protocols are permitted.

Reply:
it defaults to 1 once it gets it from the FMU. It means we don't try and start decoding a protocol until FMU tells us what to decode. Otherwise we can lock into the wrong protocol